tests/test_arrow.py

```diff
@@ -1805,3 +1805,12 @@ def test_get_iteration_params(self):
 
         with pytest.raises(ValueError):
             arrow.Arrow._get_iteration_params(None, None)
+
+
+class TestArrowPy2Deprecation:
+    @pytest.mark.skipif(
+        sys.version_info[:2] > (3, 5), reason="requires python3.6 or higher"
+    )
+    def test_py2deprecation(self):
+        with pytest.deprecated_call():
+            arrow.Arrow(2020, 12, 12)
```

Comment:
Let's test 3.6+ to make sure the warning is NOT emitted. This will ensure the coverage gets to 100%.